src/coreclr/pal/src/thread/thread.cpp

```diff
@@ -2600,6 +2600,7 @@ CPalThread::FreeSignalAlternateStack()
         // ss_size is >= MINSIGSTKSZ even in this case.
         ss.ss_size = MINSIGSTKSZ;
         ss.ss_flags = SS_DISABLE;
+        ss.ss_sp = NULL;
```

Comment:
`ss_sp` is not used when `SS_DISABLE` so there is no real issue here.
Initializing to zero to make valgrind happy.